src/utils/init-project/fsh/ig-data/input/pagecontent/index.md

```diff
@@ -0,0 +1,5 @@
+# Example SUSHI Project
+
+Welcome to the example SUSHI Project. This project is meant to help people get started using SUSHI to create an Implementation Guide.
+
+Feel free to modify this index page with your own awesome content!
```

Comment:
Since we asked the user to tell us the name of the project, can we update this file to use it instead of "Example SUSHI Project"?  It's not really an _example_...